app/models/organization.rb

```diff
@@ -402,10 +375,9 @@ def has_tag? tag
 
   private
 
-  def create_editable_statuses
-    EditableStatus.statuses.each do |status|
-      self.editable_statuses.create(status: status)
-    end
+  def create_statuses
+    EditableStatus.import EditableStatus.types.map{|status| EditableStatus.new(organization: self, status: status)}
+    AvailableStatus.import AvailableStatus.types.map{|status| AvailableStatus.new(organization: self, status: status)}
```

Comment:
This doesn't appear to be correctly creating statuses on a new organization. Another thing I noticed is that it doesn't handle the "selected" attribute on the default statuses.
<img width="749" alt="image" src="https://user-images.githubusercontent.com/12898988/32554989-8cdeec22-c469-11e7-8ae3-2c7c75dbbf99.png">

Reply:
My thought for how to set the `selected` attribute is something like this
```EditableStatus.types.map{|status| EditableStatus.create(organization: self, status: status, selected: AvailableStatus.defaults.include?(status))}```

I'm not actually sure what import does

Reply:
This is the correct behavior for when use_default_statuses is set to true for an organization.